pkgs/by-name/re/redis-dump-go/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "redis-dump-go";
+  version = "0.8.2";
+
+  vendorHash = null;
+
+  src = fetchFromGitHub {
+    owner = "yannh";
+    repo = "redis-dump-go";
+    tag = "v${version}";
+    hash = "sha256-+5iYigtMQvd6D90mpKyMa7ZKm2UDtCG91uFZ7dURBT4=";
+  };
+
```

Comment:
add nix-update-script, add versionCheckHook if functional

Reply:
There is no version argument for this program. Update script added.